doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -243,6 +243,16 @@ Amazon Sidewalk samples
 
 Bluetooth samples
 -----------------
+
+* Added support for the nRF54LM20 PDK in the following samples:
+
+   * :ref:`peripheral_lbs`
+   * :ref:`central_uart`
+   * :ref:`peripheral_uart`
+   * :ref:`peripheral_hids_mouse`
+   * :ref:`peripheral_hids_keyboard`
+   * :ref:`bluetooth_central_hids`
```

Comment:
consider reordering to respect the alphabetical order (of the real names behind the aliases)